lib/system/zephyr/cache.h

```diff
@@ -29,21 +29,39 @@
  */
 
 /*
- * @file	zephyr/cache.c
- * @brief	Zephyr libmetal cache handling.
+ * @file	zephyr/cache.h
+ * @brief	Zephyr cache operation primitives for libmetal.
  */
 
-#include <metal/cache.h>
+#ifndef __METAL_CACHE__H__
+#error "Include metal/cache.h instead of metal/zephyr/cache.h"
+#endif
 
-extern void metal_machine_cache_flush(void *addr, unsigned int len);
-extern void metal_machine_cache_invalidate(void *addr, unsigned int len);
+#ifndef __METAL_ZEPHYR_CACHE__H__
+#define __METAL_ZEPHYR_CACHE__H__
 
-void metal_cache_flush(void *addr, unsigned int len)
+#include <metal/utilities.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+static inline void __metal_cache_flush(void *addr, unsigned int len)
 {
-	metal_machine_cache_flush(addr, len);
+	metal_unused(addr);
+	metal_unused(len);
```

Comment:
Please add comments on the reason for not implementing the function.
Is it going to implement it in future?